index.html

```diff
@@ -24,22 +24,35 @@ <h1>Welcome to the Intergalactic Travel Agency.</h1>
           But what your delightful sandwich-making mother didn't bring to the table was a jar of marmite; not only the crucial last step in making your sandwhich delightful, but also central in keeping your galatactic TomTom running smoothly. Thanks Mum, instead of heading to Mars I'm barrelling towards the Sun at lightspeed.
         </h2>
-        <p>And this is where we come in.</p>
+        <p>This is where we come in.</p>
         <h3>Long gone are the days where you could simply pick up your edition of 'Hitchhiker's Guide to the Galaxy' and set about on your merry way. Oh no, the universe has changed since then; long gone are the days of typewriters, printers, IOS, and Doo-Hickey450's - Welcome to the 32nd century grandpa. If you wanna give this a shot and keep all your cyber-limbs in one piece, your going to want to speak to us.</h3>
       </section>
     <section>
       <h3>Please cycle through the above links to find the ideal hitchhiking package for you, or don't if you can't be bothered, we get it, it's been a long day.</h3>
-      <h3>If you would like to give us all your money or just the amount that is quoted under each package please click<a href="https://www.boredpanda.com/angry-kittens/">here</a>.</h3>
+      <h3>If you would like to give us all your money or just the amount that is quoted under each package please click <a href="https://www.boredpanda.com/angry-kittens/">here</a> to pay for your purchases.</h3>
       <h3>If that is the case, please submit your contact details below with any queries or questions you have or reach out to us on our satelite code: vs468 696y 236rt to speak to one of our friendly members of staff.</h3>
       <form>
         <fieldset>
           <legend>What is your reason for contacting?</legend>
-          <input id="yes" type="checkbox" name="interview?" value="yes" checked> <label for="yes">Of Course!</label>
-          <input id="unsure" type="checkbox" name="interview?" value="unsure"> <label for="unsure">Hmmmm, let me think it over...</label>
-          <input id="no" type="checkbox" name="interview?" value="no"> <label for="no">Try Again :(</label>
+          <input id="yes" type="checkbox" name="contacting" value="yes" checked> <label for="yes">I would like to purchase a package</label>
+          <input id="unsure" type="checkbox" name="contacting" value="unsure"> <label for="unsure">Please can I speak to a manager (Idiot, you're already speaking to them, I am the manager. What, you've never heard of a Staionary Cyborg Supervisor(SCS) before)</label>
+          <input id="no" type="checkbox" name="interview?" value="no"> <label for="no">I want to write a complaint*</label>
         </fieldset>
+        
+      <form id="textForm" onsubmit="submitForm(event)">
```

Comment:
I think its generally bad to have a form inside of another form; would suggest you combine these into a single form

Reply:
All done now, think I have it sorted - please check my most recent push!